.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Please remove this file since it is under `.vscode`.

Reply:
I still believe the .vscode should be included so at least people using vscode can benefit from it. It doesn't hurt if people are not using vscode right?

Reply:
Hey @marcofranssen, as much as we love VSCode, it seems common practice to not include tooling specific entries in oss efforts.

Reply:
@SteveLasker so you are expecting every potential contributor to accordingly setup his own environment. E.g. configuring the goimports to accordingly order the imports statements etc.

If people or not using vscode it doesn't hurt to have this, If they do, they have the settings in place to get them up and running very quickly and have settings that are aligned with the CI checks.

Reply:
Removed it from this PR so we can move forward. Still think we should use this to get new contributors more quickly onboarded.